src/tox/package/builder/isolated.py

```diff
@@ -103,6 +124,7 @@ def perform_isolated_build(build_info, package_venv, dist_dir, setup_dir):
                 str(dist_dir),
                 build_info.backend_module,
                 build_info.backend_object,
+                ":".join(str(p) for p in build_info.backend_paths),
```

Comment:
can we use instead the os specific variable here?

Reply:
Ah, fair enough. Though, `:` in paths is unescapable per POSIX using the standard `$PATH` delimiter.